### ISSUE TITLE
Panic: runtime error: slice bounds out of range 0.10

### ISSUE THREAD

**Issue by ruudboon** [influxdata/influxdb#5917]
When doing a query like this

SELECT time, status FROM nginx LIMIT 10

Running FreeBSD 10.2 with InfluxDB 0.10 installed from the ports.
My influxdb instance crashes with the following error.

```
[query] 2016/03/06 10:25:08 SELECT time, status FROM fluentd."default".nginx LIMIT 10
panic: runtime error: slice bounds out of range

goroutine 94 [running]:
github.com/influxdb/influxdb/tsdb.(*FieldCodec).DecodeByID(0xc8203398b0, 0xc8203cdc03, 0xc820d90940, 0x49, 0x6dd, 0x0, 0x0, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/shard.go:735 +0xa31
github.com/influxdb/influxdb/tsdb.(*FieldCodec).DecodeByName(0xc8203398b0, 0xc820596e20, 0x6, 0xc820d9091d, 0x6c, 0x700, 0x0, 0x0, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/shard.go:757 +0x13c
github.com/influxdb/influxdb/tsdb/engine/wal.DecodeKeyValue(0xc8205975f0, 0x1, 0x1, 0xc8203398b0, 0xc820d90911, 0x8, 0x70c, 0xc820d9091d, 0x6c, 0x700, ...)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:1600 +0x1b9
github.com/influxdb/influxdb/tsdb/engine/bz1.(*Cursor).read(0xc820500a80, 0x1431134f5104e400, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/engine/bz1/bz1.go:821 +0x229
github.com/influxdb/influxdb/tsdb/engine/bz1.(*Cursor).Next(0xc820500a80, 0xc820d617d8, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/engine/bz1/bz1.go:765 +0x1c4
github.com/influxdb/influxdb/tsdb.(*multiCursor).pop(0xc820d617c0, 0x1431134f5104e400, 0xa3b940, 0xc820818c90)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/cursor.go:97 +0x109
github.com/influxdb/influxdb/tsdb.(*multiCursor).Next(0xc820d617c0, 0xc8201ddd40, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/cursor.go:76 +0x2d
github.com/influxdb/influxdb/tsdb.(*TagsCursor).Next(0xc8208a3b30, 0xc8201ddd40, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/cursor.go:378 +0x7f
github.com/influxdb/influxdb/tsdb.(*TagSetCursor).Next(0xc820436900, 0x0, 0x143936e461b9ef98, 0x1431134cfcf90000, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/cursor.go:235 +0x1ad
github.com/influxdb/influxdb/tsdb.(*RawMapper).NextChunk(0xc8208772c0, 0x0, 0x0, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/raw.go:940 +0xe5
github.com/influxdb/influxdb/tsdb.(*StatefulMapper).NextChunk(0xc820772820, 0x2, 0x0, 0x0)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/mapper.go:26 +0x4d
github.com/influxdb/influxdb/tsdb.(*RawExecutor).execute(0xc82056e000, 0xc82052b020, 0xc82052af00)
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/raw.go:105 +0x17a4
created by github.com/influxdb/influxdb/tsdb.(*RawExecutor).Execute
	/usr/ports/databases/influxdb/work/influxdb-0.10.0/src/github.com/influxdb/influxdb/tsdb/raw.go:59 +0x67
```

**Comment by ruudboon** [comment]
Dropping the database and created a new one solved the issue

**Comment by seiflotfy** [comment]
Pretty sure this has been solved in master

**Comment by e-dard** [comment]
@jsternberg @benbjohnson any idea if this will be resolved for `0.11`?

**Comment by jsternberg** [comment]
This looks like an error with the bz1 engine. We've deprecated that engine in favor of tsm. You can use the `influx_tsm` tool to convert and this should go away.

Please reopen if you have any trouble. Thanks.
